app-kotlin/build.gradle

```diff
@@ -5,37 +5,46 @@ buildscript {
   }
 
   dependencies {
-    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21'
+    classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.0'
   }
 }
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-  compileSdkVersion 28
-  buildToolsVersion '28.0.3'
+  compileSdkVersion 31
 
-  defaultConfig {
+    defaultConfig {
```

Comment:
Looks like you've introduced a space here, is this intended?

Reply:
thanks for mentioning i think this is not intended.